ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLandOwnership.vue

```diff
@@ -52,15 +138,28 @@ export default defineComponent({
 
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
-.mhr-home-land-ownership::v-deep {
 
-  .ownership-checkbox {
-    label {
-      line-height: 24px;
-    }
-    .v-input__slot {
-      align-items: flex-start;
-    }
-  }
+.yes-radio {
+  width: 44%;
+  margin-right: 24px !important;
+  background-color: rgba(0, 0, 0, 0.06);
```

Comment:
We usually use hex values for colours (not rgba). Also, as Cameron mentioned, check if we already have colours defined.

Reply:
@RuoxuanPengBC

Reply:
Can you ask designers to provide the hex value for color?

Reply:
Or if too much hassle, i'm ok to resolving and moving on.

Reply:
I used the online converter to transfer rgba(0,0,0,0.06), the corresponding hex value #000000f, which is not accepted by the code base.

Reply:
Ok lets leave as is. Thanks for trying.